clouddriver-lambda/src/main/java/com/netflix/spinnaker/clouddriver/lambda/provider/agent/LambdaCachingAgent.java

```diff
@@ -113,31 +121,60 @@ public CacheResult loadData(ProviderCache providerCache) {
       }
 
       ListFunctionsResult listFunctionsResult = lambda.listFunctions(listFunctionsRequest);
+
       lstFunction.addAll(listFunctionsResult.getFunctions());
       nextMarker = listFunctionsResult.getNextMarker();
 
     } while (nextMarker != null && nextMarker.length() != 0);
 
     Collection<CacheData> data = new LinkedList<>();
+    Collection<CacheData> appData = new LinkedList<>();
+    Map<String, Collection<String>> appRelationships = new HashMap<String, Collection<String>>();
+
+    Map<String, Collection<CacheData>> cacheResults = new HashMap<>();
     for (FunctionConfiguration x : lstFunction) {
       Map<String, Object> attributes = objectMapper.convertValue(x, ATTRIBUTES);
       attributes.put("account", account.getName());
       attributes.put("region", region);
-
       attributes.put("revisions", listFunctionRevisions(x.getFunctionArn()));
       attributes.put("aliasConfiguration", listAliasConfiguration(x.getFunctionArn()));
       attributes.put(
           "eventSourceMappings", listEventSourceMappingConfiguration(x.getFunctionArn()));
-
+      // attributes.put("functionConfiguration", getFunctionConfiguration(x.getFunctionArn());
```

Comment:
can you remove this if it is unnecessary

Reply:
Deleted.